### DISCUSSION TITLE
Pure ui connect button

### DISCUSSION THREAD

**Discussion by o-az** [rainbow-me/rainbowkit#1692] [Feedback]
Is it possible to use the connect button but without any functionality, just the ui part?

**Comment by DanielSinclair** [comment]
@o-az Can you expand on what you're looking for? 🤔

**Reply by o-az** [reply]
sorry for the late reply. I meant only the ui part of the connect button so I can pass any `onClick` to it.

**Comment by 0xmkh** [comment]
@o-az There isn't a way to use connect button without any functionality unless you create your own custom connect button which you can find in our docs [here](https://www.rainbowkit.com/docs/custom-connect-button)
